botorch/acquisition/multi_objective/objective.py

```diff
@@ -262,15 +257,23 @@ def forward(self, samples: Tensor, X: Optional[Tensor] = None) -> Tensor:
         return samples * self.Y_std + self.Y_mean
 
 
-class AnalyticMultiOutputObjective(AcquisitionObjective):
+# TODO: this can't trivially be deleted because it's still used in
+# `construct_inputs_EHVI`
```

Comment:
Do we have replacement classes for this and subclasses? I vaguely recall that @SebastianAment implemented those not long ago.

Reply:
Hmm,`AcquisitionObjective` was deprecated a long time ago, but `AnalyticMultiOutputObjective` (a subclass of `AcquisitionObjective`) was only deprecated two months ago.  I'm not sure if `AnalyticMultiOutputObjective` should be considered ready for deletion then... it would have been triggering deprecation warnings for over a year in some `isinstance` checks, but not upon initialization.